pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kernelconfig/component_test.go

```diff
@@ -15,47 +15,70 @@
 package kernelconfig_test
 
 import (
+	"fmt"
+	"strconv"
+
+	"github.com/Masterminds/semver"
```

Comment:
Please move this in the bottom section of imports so that we keep it consistent.